adapter/src/hermes/adapter/constants.h

```diff
@@ -0,0 +1,20 @@
+#ifndef HERMES_ADAPTER_CONSTANTS_H
+#define HERMES_ADAPTER_CONSTANTS_H
+
+/**
+ * Constants file for all adapter.
+ * This file contains common constants across different adapters.
+ */
+
+/**
+ * HERMES_CONF env variable is used to define path to HERMES_CONF in adapters.
+ * This is used for initialization of Hermes.
+ */
+const char* HERMES_CONF = "HERMES_CONF";
```

Comment:
Could you follow the Google style for [constant names](https://google.github.io/styleguide/cppguide.html#Constant_Names)?

Reply:
Done.